configs/avae-test-config.yml

```diff
@@ -0,0 +1,51 @@
+#### Data parameters
+
+datapath: data/shrec21_full_dataset/subtomo_files
+affinity: data/shrec21_full_dataset/subtomo_files/affinity_scores.csv
+classes: data/shrec21_full_dataset/subtomo_files/classes.csv
+limit: 1000
+split: 20
+
+#### Network parameters
+
+depth:  3
+channels:  64
+latent_dims:  8
+pose_dims: 0
+
+#### Training parameters
+
+no_val_drop: true
+epochs:  10
```

Comment:
Oh, and maybe increase epoch number by default? Not that it matters much but can't train anything sensibly with 10 epochs and it might be a good idea to suggest good values here.

Reply:
fixed